src/time_zone_lookup.cc

```diff
@@ -128,6 +133,12 @@ time_zone local_time_zone() {
     if (__system_property_get("persist.sys.timezone", sysprop) > 0)
       tz_env = sysprop;
 #endif
+#if defined(__APPLE__)
+    CFTimeZoneRef system_time_zone = CFTimeZoneCopySystem();
```

Comment:
Does this `system_time_zone` need to be CFRelease()'d?

Reply:
Ah, yes! I definitely need to release that `system_time_zone` since it's copied. I just pushed a new commit which addresses this and another memory management issue.